app.controllers.js

```diff
@@ -52,5 +52,12 @@ const postCommentByArticleId = (req, res, next) => {
     })
 }
 
+const getUsers = (req, res, next) => {
+    fetchUsers().then((users) => {
+        res.status(200).send(users)
+    })
+}
```

Comment:
Controller should still have a catch block, not for any errors you reject but incase something went wrong with PSQL